jablib/src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2953,3 +2908,28 @@ You\ must\ select\ an\ identifier\ type.=You must select an identifier type.
 You\ must\ specify\ a\ Bib(La)TeX\ source.=You must specify a Bib(La)TeX source.
 You\ must\ specify\ an\ identifier.=You must specify an identifier.
 You\ must\ specify\ one\ (or\ more)\ citations.=You must specify one (or more) citations.
+
+# CommandLine
+Available\ export\ formats\:=Available export formats:
+Available\ import\ formats\:=Available import formats:
+Cannot\ embed\ metadata\ on\ any\ linked\ files\ of\ %s.\ Make\ sure\ there\ is\ at\ least\ one\ linked\ file\ and\ the\ path\ is\ correct.%n=Cannot embed metadata on any linked files of %s. Make sure there is at least one linked file and the path is correct.%n
```

Comment:
`%n` at the end is strange. We also switched to `\n` for newlines in the translation. Please either change to `\n` or just remove the `%n` at the end (and do the newline internally)